docs/library/machine.Pin.rst

```diff
@@ -242,7 +242,7 @@ The following methods are not part of the core Pin API and only implemented on c
 
    Toggle output pin from "0" to "1" or vice-versa.
 
-   Availability: mimxrt, samd, rp2 ports.
+   Availability: cc3200, esp32, esp8266, mimxrt, nrf, pic16bit, renesas-ra, rp2, samd, stm32 ports.
```

Comment:
nrf, pic16bit, renesas-ra and stm32 do not have the `Pin.toggle` method (but they do have `LED.toggle`, I think that's what you saw when you did the grep).  So please remove them from this list.

Reply:
Done.